components.go

```diff
@@ -4,13 +4,20 @@ import (
 	"fmt"
 	"strings"
 	"time"
+
+	"github.com/teambition/rrule-go"
 )
 
 // Calendar is the top-level iCalendar object.
 type Calendar struct {
 	*Component
 }
 
+// Returns the Recurrence Set for this component.
+func (comp *Component) RecurrenceSet(loc *time.Location) (*rrule.Set, error) {
+	return comp.Props.RecurrenceSet(loc)
```

Comment:
Still not a fan of having `RecurrenceSet` duplicate across `Props` and `Component`. I think it'd make more sense to only keep it in `Component`, since all `Props` helpers only operate on a single prop and are just data type helpers.

Reply:
Yeah, I was hesitant on the same thing.